docker-compose-sgx.yaml

```diff
@@ -33,6 +33,8 @@ services:
       # Below volume mappings are required for DISPLAY settings by heart disease GUI client
       - /tmp/.X11-unix:/tmp/.X11-unix
       - ~/.Xauthority:/root/.Xauthority
+      # Map aesm service on host to docker, required only in HW mode.
```

Comment:
can you remove http_proxy, https_proxy, no_proxy arguments. In Dockerfile.tcf-dev, these variables are not being read from here.

Reply:
yes, proxy variables are actually read from /etc/environment